### PR TITLE
feat: show export host target

### PR DESCRIPTION
**Pull request by TomChv**
Update directory export resolver to log additional context about the export. It logs a simple message: `export directory <name of the dir> to host <path>`.

EDIT: also include this change for file export

Resolves: #5084

### CODE REVIEW

**Comment by TomChv** [comment]
Result on SDK logging


Screenshot 2023-08-15 at 15 46 09

Result on TUI logging

Screenshot 2023-08-15 at 15 45 45

**Comment by TomChv** [comment]
Rust SDK test are failing again for a reason not related to my PR :/

**Comment by TomChv** [comment]
#5626 Shall fix this issue, I'm rebasing

**Comment by vito** [comment]
@TomChv I think it's worth showing. :+1: It's in many ways the 'primary output' and it can take a while to run.

File: core/directory.go [hunk 1]
```diff
@@ -596,6 +599,15 @@ func (dir *Directory) Export(
 		defPB = dir.LLB
 	}
 
+	rec := progrock.RecorderFromContext(ctx)
+
+	vtx := rec.Vertex(
+		digest.Digest(identity.NewID()),
+		strings.Join([]string{"export directory", dir.Dir, "to host", destPath},
```
Comment (vito) [inline core/directory.go:606]: nit: `fmt.Sprintf` would be more conventional here, but really splitting hairs

**Review by vito — APPROVED** [review]
lgtm!

File: core/directory.go (hunk 1 above)
Comment (TomChv) [reply core/directory.go:606]: You're right! I'm fixing that and merging

**Pull request merged**